server/src/ClientToDBAPI/Address/mainRoute.ts

```diff
@@ -9,7 +9,7 @@ import {GET_ALL_CITIES, GET_CITY_STREETS, GET_ALL_COUNTRIES} from '../../DBServi
 const addressRoute = Router();
 
 addressRoute.get('/cities', (request: Request, response: Response) => {
-  graphqlRequest(GET_ALL_CITIES).then((result: any) => {
+  graphqlRequest(GET_ALL_CITIES, request.headers).then((result: any) => {
```

Comment:
get request dont require saving the updator

Reply:
every graphql request needs the headers, because i only check if it is a query or mutation in the funciton